events/syscall/cron_job_schedule.go

```diff
@@ -0,0 +1,25 @@
+// +build linux
+
+package syscall
+
+import (
+	"github.com/falcosecurity/event-generator/events"
+
+	"os/exec"
+)
+
+var _ = events.Register(CronJobSchedule)
+
+func CronJobSchedule(h events.Helper) error {
+	// This just lists crons, but sufficies to trigger the event
+	// Cron detection is not enabled by default, see `consider_all_cron_jobs` in rules.yaml
+	cmd := exec.Command("/usr/bin/crontab", "-l")
+	err := cmd.Run()
+
+	// silently ignore crontab exit status 1
+	if exitErr, isExitErr := err.(*exec.ExitError); isExitErr {
```

Comment:
Since `/usr/bin/crontab` might be not present on the OS, I think we should skip the error when it is not present too.

Reply:
I used lookup path and if that fails just a `return nil`